azure-cron.yml

```
@@ -181,9 +184,28 @@ jobs:
             cd compiler/daml-extension
             sed -i "s/__VERSION__/$GITHUB/" package.json
             # This produces out/src/extension.js
-            bazel run --run_under="cd $PWD &&" @yarn//:yarn install
-            bazel run --run_under="cd $PWD &&" @yarn//:yarn compile
-            bazel run --run_under="cd $PWD &&" @daml_extension_deps//vsce/bin:vsce -- publish --yarn $GITHUB -p $MARKETPLACE_TOKEN
+
+            # Some branches have updated vsce, and different daml_extension_deps name, so we detect that.
+            # We build conditions here, rather than using a checked in script, as we're currently stuck trying
+            # to push a release to marketplace which is already frozen in git.
+
+            # Find the external deps repo name
+            if bazel query @daml_extension_deps_2x//... > /dev/null 2>& 1; then 
```

Generally we don't leave spaces between redirects. Is there a reason we want to remove the standard output? I usually find this to be really problematic if anything breaks later and we have no logs.
```suggestion
            if bazel query @daml_extension_deps_2x//... >/dev/null 2>&1; then 
```

---

The output of this command is massive, i'll add some logs around so we know whats happening